modules/Blockchain.js

```diff
@@ -1035,6 +1037,48 @@ class Blockchain {
         }
         return this.blockchain[0].getIdentity();
     }
+
+    /**
+     * Returns wallet public and private key from configuration
+     */
+    getWallet(blockchain_id) {
+        if (blockchain_id) {
```

Comment:
This code should be merged with Uros PR first, as he handles blockchain implementation in different way